Python/Product/Cookiecutter/CookiecutterToolWindow.cs

```diff
@@ -209,21 +209,19 @@ sealed class CookiecutterToolWindow : ToolWindowPane, IVsInfoBarUIEvents {
             _uiShell.UpdateCommandUI(0);
         }
 
-        private void OpenGeneratedFolder(string folderPath) {
-#if DEV15_OR_LATER
-            OpenInSolutionExplorer(folderPath);
-#else
-            OpenInWindowsExplorer(folderPath);
+        private void ExecuteCommand(string name, string args) {
+            try {
+                _dte.ExecuteCommand(name, args ?? string.Empty);
+            } catch (Exception ex) when (!ex.IsCriticalException()) {
+#if !DEV15_OR_LATER
+                if (name == "File.OpenFolder") {
```

Comment:
It's a special case for VS 2015, which we don't officially support. I don't know the state of the open folder support extension on VS 2015.  I figured let it try, in case it's there.